### DISCUSSION TITLE
12v Battery Percentage

### DISCUSSION THREAD

**Discussion by PSXDupe** [Hyundai-Kia-Connect/kia_uvo#742] [Q&A]
The last version of the integration seems to be missing the 12v Battery Percentage 

Is this a bug or has it been removed for a specific reason 

Thanks in advance

**Comment by cdnninja** [comment]
No change on our end related to battery.   I have heard of it before where it comes and goes in the native mobile app too.

**Reply by PSXDupe** [reply]
Doesn't seem to be working in mobile app or desktop browser.

**Reply by cdnninja** [reply]
If the Hyundai Kia app isn't working not much we can do.

**Reply by cdnninja** [reply]
Will need the data data sensor attributes to see if 12v is in that raw data or not.

**Comment by PSXDupe** [comment]
The Hyundai app is working. The integration is not showing the 12v Battery percentage in HA app or HA via browser.

**Reply by cdnninja** [reply]
Since confirming the native app works have you restarted the integration?

**Comment by PSXDupe** [comment]
Yes, deleted it and reinstalled.

**Comment by PSXDupe** [comment]
Previously when I installed the integration there were 48 entities,  now when I install it there are only 46. 12v battery is not there. Surely that is not related to the data coming in, at that stage you would not have even configured it to pull data

**Reply by cdnninja** [reply]
The sensors are created dynamically based on what data is provided at boot up.   If no data no sensor.

**Reply by PSXDupe** [reply]
From what I have seen over the last couple of days

If the batState = 1 then there is no batSoc in the data. If batState = 0, then the batSoc is in the data.

Do you know what batState means ?

Thanks a million

**Reply by cdnninja** [reply]
No not sure.

**Comment by wkearney99** [comment]
I have also noticed that 12v data is not being reported.  Though I'm using this via the Home Assistant integration.

And today my EV6 had to get towed to the dealer because the 12v battery died.  Hadn't driven it in about a week, but had it connected to an L1 charger.  Main battery was 100% but it let the 12v drop to 3.5vdc.  Hopefully service will sort it out.

**Reply by cdnninja** [reply]
https://github.com/Hyundai-Kia-Connect/kia_uvo/issues/719

**Comment by PSXDupe** [comment]
Same thing happened to my IONIQ 5, has never happened in the last 18 months, but on Friday last, car was dead, traction battery at 80% and plugged in to a home charger.

**Comment by nis** [comment]
Before I started using Home Assistant and this integration, I noticed that whenever the 12V battery was starting to fail, it stopped showing up in the app. After replacing the 12V battery it started showing up again.

So, somewhere, either in the car or in Hyundais systems, there is a switch that chooses to show the 12V battery status, or not.
